client/components/account-card.jsx

```diff
@@ -76,6 +87,108 @@ export default class AccountCard extends React.Component {
     return <h1> {this.state.currentRating}</h1>;
   }
 
+  handleXClick() {
+    this.setState({ toggleStarSystem: true });
+  }
+
+  handleCheckClick() {
+    const form = new FormData();
+    form.append('whoRated', this.state.userLoggedIn);
+    form.append('ratedWho', this.props.username);
+    form.append('rating', this.state.ratingValue / 2);
+    const requestObj = {
+      method: 'POST',
+      body: form
+    };
+    fetch('/api/uploads/ratings', requestObj)
+      .then(result => result.json())
+      .then(result => {
+        this.setState({
+          ratingValue: 0,
+          ratingClicked: false
+        });
+
+        const requestObj3 = {
+          method: 'GET'
+        };
+        fetch(`/api/accounts/new-ratings/${this.props.username}`, requestObj3)
+          .then(result => result.json())
+          .then(result => {
+            if (result.updatedRating) {
+              this.setState({
+                currentRating: result.updatedRating
+              });
+              const requestObj2 = {
+                method: 'PUT'
+              };
+              fetch(`/api/accounts/new-rating/${this.props.username}&${result.updatedRating}`, requestObj2)
+                .then(result => result.json())
```

Comment:
In most situations, when we have to keep talking back and forth to the server like this, our server is being too lazy and not doing enough work for us.